lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -1193,11 +1197,17 @@ def install_puppet_agent_dev_repo_on( hosts, opts )
               raise "No repository installation step for #{variant} yet..."
             end
 
-            onhost_copied_file = File.join(onhost_copy_base, release_file)
-            fetch_http_file( release_path, release_file, copy_dir_local)
-            scp_to host, File.join(copy_dir_local, release_file), onhost_copy_base
+            if host['platform'] =~ /eos/
```

Comment:
I agree.  I thought about making a `get_remote_file` file in the `unix/file.rb` code so that that could just be `host.get_remote_file` rather than the if statement, but I backed down from that :-/